projects/schematics/src/add/index.spec.ts

```diff
@@ -59,6 +60,18 @@ describe('ng add @clr/angular', () => {
       expect(styles.includes('node_modules/@clr/ui/clr-ui.min.css')).toBeTruthy();
     });
 
+    it('should add return Clarity version as correct', async () => {
+      const correct = getVersion('^12.0.0', '12.0.0-next.0');
+
+      expect(correct).toEqual('12.0.0-next.0');
+    });
+
+    it('should return Latest Clarity version', async () => {
+      const correct = getVersion('^12.0.0', '13.0.0');
+
+      expect(correct).toEqual('^12.0.0');
+    });
+
```

Comment:
Add a test that checks for Angular 11, 10 that you get Clarity 5, 4, respectively.